pkg/docker/config/config_test.go

```diff
@@ -636,6 +637,42 @@ func TestGetAllCredentials(t *testing.T) {
 		require.Equal(t, d.password, conf.Password, "%v", d)
 	}
 
+	// test "credStore"
+	f, err := os.OpenFile(authFilePath, os.O_RDWR|os.O_TRUNC, 0644)
+	require.NoError(t, err)
+	_, err = f.Write([]byte(`{ "credsStore": "helper-registry" }`))
+	require.NoError(t, err)
+	err = f.Close()
+	require.NoError(t, err)
+	sys = types.SystemContext{
+		AuthFilePath:                authFilePath,
+		SystemRegistriesConfPath:    "",
+		SystemRegistriesConfDirPath: "",
```

Comment:
This is (sadly?) the same as leaving the values unset, in particular it doesn’t disable any ordinary users’ configuration. So if this data matters, it should point at real files.

Reply:
@mtrmac Do I need to set it? I think that for this test `AuthFilePath` is enough, isn't it?